### ISSUE TITLE
Unique name error message when a page is named the same as a question

### ISSUE THREAD

**Issue by mgoetz-nerdery** [surveyjs/survey-creator#5896]
### Are you requesting a feature, reporting a bug or ask a question?

Reporting a bug

### What is the current behavior?

If you rename a question so that it has the same name as a page or vice versa, you will receive the "Please enter a unique name" error message.

### What is the expected behavior?

The "please enter a unique name" error only appears if two pages are named the same, or two questions are named the same.

### How would you reproduce the current behavior (if this is a bug)?

Rename a page so that it has the same name as a question.  Notice that the "Please enter a unique name" text appears.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

https://surveyjs.io/survey-creator/examples/free-nps-survey-template/reactjs

### Specify your

* browser: Google Chrome 128
* editor version: latest (as of September 16, 2024)

**Comment by andrewtelnov** [comment]
@mgoetz-nerdery Do you have the real user case here? We have the code where you need to find out the root element by it's name without element type (page, panel or question).

Thank you,
Andrew

**Comment by mgoetz-nerdery** [comment]
@andrewtelnov I just used one of the creator demos: https://surveyjs.io/survey-creator/examples/free-nps-survey-template/reactjs

**Comment by andrewtelnov** [comment]
@mgoetz-nerdery Yes, I know that it can be reproduced easily. My point is that the current behavior is to have unique names across all elements: questions, panels, and pages. This allows us to find an element by name without knowing its type. 

You want to change this behavior. My question is, why do you want to do it? What is your scenario for wanting to name a page and a question the same? What is the benefit of changing the current functionality?

Thank you,
Andrew

**Comment by mgoetz-nerdery** [comment]
@andrewtelnov Thanks for the insight.  I did speak to my stakeholders and this is not really a priority.  I'll close this issue.  Thanks!

**Comment by andrewtelnov** [comment]
You are very welcome!

Andrew
